extensions/amp/bundle/pom.xml

```diff
@@ -53,6 +53,15 @@
                             <supportIncrementalBuild>true</supportIncrementalBuild>
                         </configuration>
                     </execution>
+                    <execution>
```

Comment:
Shouldn't the baseline check be enabled?

Reply:
This is needed to overcome a previous mistake: deploying `com.adobe.cq:core.wcm.components.extensions.amp:2.8.1-SNAPSHOT-20200410121300` as a release.

Reply:
Will remove it after we do a proper release

Reply:
Done in #1100